dockerfiles/alpine/obiam/README.md

```diff
@@ -29,9 +30,9 @@ git clone https://github.com/wso2/docker-open-banking.git
 
 - Navigate to `<OBIAM_DOCKERFILE_HOME>` directory. <br>
   Execute `docker build` command as shown below.
-    + `docker build --build-arg WSO2_SERVER_DIST_URL=<URL_OF_THE_HOSTED_LOCATION/FILENAME> --build-arg WSO2_OB_KEYMANAGER_DIST_URL=<URL_OF_THE_HOSTED_LOCATION/FILENAME> --build-arg WSO2_OB_CERTS_URL=<URL_OF_THE_HOSTED_LOCATION/FILENAME> -t wso2-obiam:2.0.0-alpine .` <br>
-    > eg:- **Hosted locally**: `docker build --build-arg WSO2_OB_Accelerator_DIST_URL=http://localhost:8000/wso2-obiam-accelerator-3.0.0.tar.gz --build-arg WSO2_OB_KEYMANAGER_DIST_URL=http://localhost:8000/wso2is-extensions-1.2.10.tar.gz --build-arg WSO2_OB_CERTS_URL=http://localhost:8000/ob-cert.zip  -t wso2-obiam:2.0.0-alpine .` <br><br>
-      eg:- **Hosted remotely**: `docker build --build-arg WSO2_OB_Accelerator_DIST_URL=http://<public_ip:port>/wso2-obiam-accelerator-3.0.0.tar.gz --build-arg WSO2_OB_KEYMANAGER_DIST_URL=http://<public_ip:port>/wso2is-extensions-1.2.10.tar.gz --build-arg WSO2_OB_CERTS_URL=http://<public_ip:port>/ob-cert.zip  -t wso2-obiam:2.0.0-alpine .`
+    + `docker build --build-arg WSO2_OB_Accelerator_DIST_URL=<URL_OF_THE_HOSTED_LOCATION/FILENAME> --build-arg WSO2_OB_KEYMANAGER_DIST_URL=<URL_OF_THE_HOSTED_LOCATION/FILENAME> --build-arg OB_TRUSTED_CERTS_URL=<URL_OF_THE_HOSTED_LOCATION/FILENAME> --build-arg WSO2_OB_KEYSTORES_URL=<URL_OF_THE_HOSTED_LOCATION/FOLDER_NAME> -t wso2-obiam:2.0.0-alpine .` <br>
```

Comment:
The tag is incorrect. Should be changed to 3.0.0

Reply:
changed it https://github.com/wso2/docker-open-banking/pull/149/commits/89b83d64b61cb15abe1b1f4ea239a556a28f5fbc